src/nih_wayfinding/app/scripts/views/navigate/navigate-partial.html

```diff
@@ -9,7 +9,7 @@
 <footer class="container-fluid">
     <nav class="row">
         <div class="btn-group-nav btn-group-justified">
-            <a ng-click="navigate.prevStep()" type="button" class="btn btn-primary btn-block">Previous Step</a>
+            <a ng-click="navigate.offCourse()" type="button" class="btn btn-primary btn-block">Off-Course</a>
```

Comment:
This is what @jbranigan suggested during scrum today. Will be chatting with @jfrankl in a bit, we should have a better long term plan after that.